plugin-server/src/main/ingestion-queues/session-recordings-consumer.ts

```diff
@@ -135,41 +135,24 @@ export const eachBatch =
 
             let teamId: number | null = null
 
-            try {
-                teamId =
-                    messagePayload.team_id ?? (event.token ? (await teamManager.getTeamByToken(event.token))?.id : null)
-
-                if (!teamId) {
-                    status.warn('⚠️', 'invalid_message', {
-                        reason: 'team_not_found',
-                        partition: batch.partition,
-                        offset: message.offset,
-                    })
-                    await producer.queueMessage({ topic: KAFKA_SESSION_RECORDING_EVENTS_DLQ, messages: [message] })
-                    continue
-                }
+            teamId =
+                messagePayload.team_id ?? (event.token ? (await teamManager.getTeamByToken(event.token))?.id : null)
```

Comment:
this part should be inside the try-catch block probably still - so if Postgress is unavailable we retry

Reply:
Interestingly, adding the try-catch will reduce the amount it will retry. KafkaJS will try on any error thrown I think. I'll reverted to how it was before though as I haven't meant to change this!